### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by fatmatanis**
navbar

### CODE REVIEW

File: src/components/Navbar/_search.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { INavbarProps } from "../../types/types";
+
+const Search = ({ styleName }: INavbarProps) => {
+  return <input type="search" placeholder="Search" className={styleName} />;
```
Comment (atvdi) [inline src/components/Navbar/_search.tsx]:
```suggestion
const Search = ({ className }: INavbarProps) => {
  return <input type="search" placeholder="Search" className={className} />;
```

File: src/components/Navbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import Search from "./_search";
```
Comment (atvdi) [inline src/components/Navbar/index.tsx]: Why the component name is starting with underscore?

File: src/components/Navbar/styles.scss [hunk 3]
```diff
@@ -0,0 +1,112 @@
+.nav {
+    width: 100%;
+    height: 88px;
+    margin: 0;
+    padding: 0;
```
Comment (atvdi) [inline src/components/Navbar/styles.scss]: It depends on the situation but I think we shouldn't use fixed heights. Font size and vertical spacing should determine the height.

**Review by atvdi — CHANGES_REQUESTED** [review]

File: src/components/Navbar/index.tsx (hunk 2 above)
Comment (fatmatanis) [reply src/components/Navbar/index.tsx]: I got the suggestion that it is better to use. to indicate that I will use it in the with other component. but actually I didn't understand the basic difference. I will change it, thanks

File: src/components/Navbar/styles.scss (hunk 3 above)
Comment (fatmatanis) [reply src/components/Navbar/styles.scss]: I will try this, thank you

**Comment by fatmatanis** [comment]
navbar

**Review by iremlaya — APPROVED** [review]

**Review by atvdi — APPROVED** [review]

**Pull request merged**